includes/create-theme/theme-styles.php

```diff
@@ -26,15 +26,18 @@ public static function update_style_css( $style_css, $theme ) {
 		$author_uri    = $theme['author_uri'];
 		$wp_version    = CBT_Theme_Utils::get_current_wordpress_version();
 		$wp_min        = $current_theme->get( 'RequiresWP' );
-		$version       = $theme['version'];
-		$requires_php  = $current_theme->get( 'RequiresPHP' );
-		$text_domain   = $theme['slug'];
-		$template      = $current_theme->get( 'Template' ) ? "\n" . 'Template: ' . $current_theme->get( 'Template' ) : '';
-		$license       = $style_data['License'] ? $style_data['License'] : 'GNU General Public License v2 or later';
-		$license_uri   = $style_data['LicenseURI'] ? $style_data['LicenseURI'] : 'http://www.gnu.org/licenses/gpl-2.0.html';
-		$tags          = CBT_Theme_Tags::theme_tags_list( $theme );
-		$css_contents  = $css_contents ? "\n\n" . $css_contents : '';
-		$copyright     = '';
+		if ( ! $wp_min ) {
```

Comment:
We should probably use this same logic in `build_style_css` below, which builds the style.css for child themes.

We're also hard-coding `6.0` for the "Requires at least" value in the readme files on [line 70 of theme-readme.php](https://github.com/WordPress/create-block-theme/blob/trunk/includes/create-theme/theme-readme.php#L70), so maybe we could use the same logic here too.